docs/source/guide/pt-1-intro.md

```diff
@@ -50,56 +52,84 @@ the circuit on a noisy simulator, and returns the probability of the ground
 state. See the [Executors](executors.md) section for more information on
 how to define more advanced executors.
 
+During execution by the simulator, a coherent error is introduced in the circuit
+by applying a rotation around the X-axis (Rx gate) to each output of any 2-qubit gate.
+
+This noise model is well-suited to highlight the effect of Pauli Twirling,
+which is a technique that turns coherent noise into incoherent Pauli channels.
+
+For the sake of this example, we define the noise level as the angle of the Rx rotation.
```

Comment:
```suggestion
For the sake of this example executed by a simulator, we define the noise level as an Rx rotation by some angle.
```

Move this line to before L55

Reply:
I moved this line up, and changed the meaning of noise_level, take a look at commit 37258d62b113ea2513e124b0acb56055ce421d96

Reply:
LGTM! Feel free to merge.